internal/datanode/data_sync_service.go

```diff
@@ -170,7 +170,7 @@ func (dsService *dataSyncService) initNodes(vchanInfo *datapb.VchannelInfo) erro
 				pos:     *us.GetDmlPosition(),
 			}
 		}
-		if err := dsService.replica.addNormalSegment(us.GetID(), us.CollectionID, us.PartitionID, us.GetInsertChannel(), us.GetNumOfRows(), us.Statslogs, cp); err != nil {
```

Comment:
Please add a log in L164 for checkpoint timestamp

Reply:
Have discussed.